Azure/sonic-buildimage
# [multi-asic][sonic-config-engine]: Get PORT table from namespace config db

Signed-off-by: Suvarna Meenakshi <sumeenak@microsoft.com>



#### Why I did it
portconfig.py gets PORT table from config_db if it is present. If not, port_config.ini files are parsed.
For multi-asic platform, if namespace is passed to get_port_config(), config_db connection was done to host namespace always and not to asic specific namespace.
Provides fix for: https://github.com/Azure/sonic-buildimage/issues/7161

#### How I did it
Modify db connection function to connect to namespace config_db.

#### How to verify it
Bring up multi-asic VS switch:
./testbed-cli.sh -t vtestbed.csv -m veos_vtb -k ceos add-topo vms-kvm-four-asic-t1-lag password.txt
./testbed-cli.sh -t vtestbed.csv -m veos_vtb deploy-mg vms-kvm-four-asic-t1-lag lab password.txt 

Ensure that 'config load_minigraph' is successful and and config_db*.json are generated correctly.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
